tests/testthat/test-cache.R

```diff
@@ -24,6 +24,15 @@ test_that("reads from and writes to cache", {
   expect_equal(as.character(css), expected)
 
   expect_equal(sass_cache_get()$size(), 1)
+
+  # Modifying the file busts the cache (even it if has the same contents)
```

Comment:
If you just need to change the mtime of a file for the test, you can do something like:

```R
Sys.setFileTime(filename, Sys.time() + 5)
```

I used `+ 5` just to be safe.